packages/js/product-editor/src/components/notice/style.scss

```diff
@@ -0,0 +1,22 @@
+.woocommerce-product-notice {
+	padding: $gap-small $gap;
+	p {
+		color: $gray-900;
+		letter-spacing: 0.24px;
+	}
+	button {
+		letter-spacing: 0.24px;
+	}
+	&.info {
+		background-color: #f0f6fc;
+	}
+	&.success {
+		background-color: $studio-green-5;
+	}
+	&.error-type {
+		background-color: $studio-red-5;
+	}
+	&.warning {
+		background-color: $studio-yellow-5;
```

Comment:
Maybe for success, error, and warning we want to use the same colour settings as the original notice: https://github.com/WordPress/gutenberg/blob/trunk/packages/components/src/notice/style.scss#L16-L29
As it isn't that far off from the design ( cc: @jarekmorawski it would be nice if we can keep the notice background colours the same as the Gutenberg notice: https://wordpress.github.io/gutenberg/?path=/docs/components-notice--docs )

Reply:
That's correct. @octaedro, can we use the styling from the Figma I shared with you yesterday on Slack? Not just colors, but also paddings and font sizes. It'll take a few more hours but if we get it right, we could make it a component to use across all Woo products.

Reply:
Thank you @jarekmorawski and @louwie17
Since the colors I used are an addition out of scope for this PR. What do you think about opening a new issue to tackle that later? I deleted the styles related to the colors in the commit [698411a](https://github.com/woocommerce/woocommerce/pull/39675/commits/698411a7bd49c0697e76e7367ea52bda381dfc07)